jt-classbreaks/src/main/java/it/geosolutions/jaiext/classbreaks/HistogramClassification.java

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
we may need to add the jai-ext license here too

Reply:
Good catch, I'll add it